SU-SWS/stanford_profile_helper
# D8CORE-3052: Centering the title and button for teasers.

# READY FOR REVIEW

# Summary
- This centers the button and the title for the teaser paragraphs

# Needed By (Date)
- 1/12

# Urgency
- low

# Steps to Test

1. pull in the changes
2. Create teasers and verify the titles and buttons are centered .

# Affected Projects or Products
- Stanford Basic 

# Associated Issues and/or People
- D8CORE-3052

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
